packages/ocaml-lua/ocaml-lua.1.7/opam

```diff
@@ -0,0 +1,59 @@
+opam-version: "2.0"
+maintainer: "p.donadeo@gmail.com"
+authors: [
+  "Paolo Donadeo"
+  "Sylvain Le Gall"
+]
+homepage: "https://pdonadeo.github.io/ocaml-lua/"
+bug-reports: "https://github.com/pdonadeo/ocaml-lua/issues"
+dev-repo: "git+https://github.com/pdonadeo/ocaml-lua.git"
+license: "MIT"
+build: [
+  [
+    "./configure"
+    "--enable-docs"
+    "--prefix"
+    prefix
+    "--docdir"
+    "%{doc}%/ocaml-lua"
+    "--libdir"
+    "%{lib}%/ocaml-lua"
+  ]
+  [make]
+  [make "doc"]
+]
+remove: [
+    ["ocamlfind" "remove" "lua"]
+    ["rm" "-R" "%{doc}%/ocaml-lua"]
+]
+depends: [
+  "ocaml" {>= "4.02.1" & < "4.08.0"}
+  "ocamlfind"
```

Comment:
I can't see any use of the `Findlib` library in your code. Could you add the `{build}` attribute if you don't use it?

Reply:
findlib is actually used to install and remove the library, I'm going to add the `{build}` constraint.

Reply:
About OSX: is the output file `~/.opam/log/ocaml-lua-34869-a71bb4.out` available for a check? It seems the configure script can't write in `/tmp` but I'm not sure at all.